### ISSUE TITLE
'error clear' reported on each template closing tag in all components

### ISSUE THREAD

**Issue by violoncelloCH** [nuxt/eslint-plugin-nuxt#121]
with version 2.0.0 of nuxt/eslint-plugin-nuxt I get a error for every component's template closing tag; 
the lint errors all look like this: 
```
/home/jo/repos/xxx/xxxxx/components/blog/blogcard.vue
  14:12  error  clear  vue/comment-directive
```
the reported lines contain the end template tag:
```
</template>
```

from a quick look at the code (https://github.com/vuejs/eslint-plugin-vue/blob/80b8983118cc08cf473642c05413bcb930c93613/lib/rules/comment-directive.js#L340) it looks to me that the message clear gets reported to the post process to say that no error was found, however... I'm not sure if this is an issue with eslint-plugin-nuxt or exlint-plugin-vue directly? please advise

**Comment by violoncelloCH** [comment]
okay seems like updating one of those:
@nuxtjs/eslint-config to v5
@nuxtjs/eslint-module to v3.0.1
has fixed the issue... 
sorry for the noise

**Comment by GuilhermeArsenio** [comment]
This still happens to me with:
@nuxtjs/eslint-config @v6.0.0
@nuxtjs/eslint-module @v3.0.1
@eslint @v7.22.0
